build.gradle

```diff
@@ -68,13 +68,17 @@ dependencies {
     compile group: 'commons-validator', name: 'commons-validator', version: '1.6'
     compile group: 'commons-io', name: 'commons-io', version: '2.6'
     compile group: 'uk.gov.hmcts.reform.auth', name: 'auth-checker-lib', version: '2.1.3'
-    compile group: 'uk.gov.hmcts.reform', name: 'service-auth-provider-client', version: '2.0.0'
+    compile group: 'uk.gov.hmcts.reform', name: 'idam-client', version: '0.0.5'
     compile group: 'uk.gov.hmcts.reform', name: 'java-logging-appinsights', version: '5.0.1'
+    compile group: 'uk.gov.hmcts.reform', name: 'java-logging-spring', version: '3.0.4'
+    compile group: 'uk.gov.hmcts.reform', name: 'service-auth-provider-client', version: '2.0.0'
     compile "org.mapstruct:mapstruct-jdk8:1.3.0.Final"
     annotationProcessor "org.mapstruct:mapstruct-processor:1.3.0.Final"
     annotationProcessor "org.springframework.boot:spring-boot-configuration-processor"
 
-    testCompile "org.springframework.boot:spring-boot-starter-test"
+    testCompile('org.springframework.boot:spring-boot-starter-test') {
+        exclude group: "com.vaadin.external.google", module:"android-json"
```

Comment:
fix a json lib warning

Reply:
TimJ previously said we don't need java-logging-spring. Do we know for sure that we need it now?

Reply:
seems to be required, will chase it up later